homework/Util.js

```diff
@@ -18,18 +18,17 @@ class Util {
 
   static fetchJSON(url) {
     return new Promise((resolve, reject) => {
```

Comment:
You no longer need to create a promise yourself when you use `fetch()` because it already returns a promise itself. You only need to use `new Promise(...)` if you need to convert a callback-based into a promised based one. Can you try it without `new Promise(..)`?

Reply:
Hi @Nadoshka, I would still like to ask you to try and make this change. See it as another opportunity to practice your skills and to get feedback.

Reply:
Thanks for the explanation! I appreciate it! 
Change is done!  Thank u in advance for your feedback!